lib/md055.js

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+
+"use strict";
+
+const { addErrorContext } = require("../helpers");
+const { filterByTypes } = require("../helpers/micromark.cjs");
+
+const noPipeErrorMessage = "Table row is missing a leading or trailing pipe: ";
```

Comment:
This message duplicates the name of the rule and can be removed. The context of a violation is usually just the violation, `row.text` as you have it below.